src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp

```diff
@@ -47,17 +47,8 @@ namespace OpenMS
     { // all zero
       return .0;
     }
-    //const std::vector<double> weights({ 1.492, -2.0041, -14.3891, -0.9853, 0.4568, 0.063, 14.4072});
     const std::vector<double> weights({ -.0941, -1.9804, -12.7522, 0.2622, -1.2431, 0.0815, 13.5244});
 
-    //ChargeCos         1.492
-    //ChargeSNR       -2.0041
-    //Cos            -14.3891
-    //SNR             -0.9853
-    //ChargeScore      0.4568
-    //AvgPPMerror       0.063
-    //Intercept       14.4072
-
     // ChargeCos           0.0941
```

Comment:
these numbers don't match the ones in the weights (e.g. sign)

Reply:
please double check... this could be a bug

Reply:
@KyowonJeong would it be okay to delete these lines?

Reply:
note that the sign of cos is negative in the weights vector but positive in the comment.
on the other hand the sign of chargecos is positive in the weights vector but also positive in the comment.

Reply:
Removed. THanks!